### DISCUSSION TITLE
Would Ash be impacted by removing 'pipeline' attributes from vk.xml?

### DISCUSSION THREAD

**Discussion by filnet** [ash-rs/ash#444] [Q&A] [answered]
Came across this : https://github.com/KhronosGroup/Vulkan-ValidationLayers/issues/2834

Will that change to vk.xml impact Ash ?

**Comment by MarijnS95** [comment]
@filnet That looks a link into your private mail folder - what exactly are you referring to?

**Reply by filnet** [reply]
I fixed the link.

**Reply by MarijnS95** [reply]
Oh right, that issue is closed and explains why I couldn't find it!

(Sorry for the edit-move, trying out the Q&A thing of GH-discussions!)

**Comment by MarijnS95** [comment, marked as answer]
No, this doesn't impact Ash at all. We don't read this attribute and placing the [shared `vk.xml`](https://github.com/KhronosGroup/VK-GL-CTS/files/6536053/vk.zip) over a local `.179` regeneration yields an empty diff.

Thanks for bringing this to our attention just in case!

**Reply by filnet** [reply]
Great!

We can close this discussion then. But I can't find a close or equivalent button...

**Reply by MarijnS95** [reply]
Discussions cannot be closed, they (if in the `Q&A` category) can only have a "selected answer". One can always add new threads, and maintainers can lock conversations if they get out of hand.
